### PR TITLE
QData and QHeader implementation

### LINKED ISSUE #3: QData class
```
class QData():
    QData.header (based on ismrmrd) -> check in functions if certain parameters available
    QData.data: torch.Tensor 5D: [everything_else, coil, z, y, x]

   from_kdata()
   from_idata()
   from_ismrmrd()
   from_nifty()
   from_dicom()
```

Check list:
- [ ]     Create branch
- [ ]     Implement code
- [ ]     Add comments
- [ ]     Implement tests
- [ ]     Squash commmits
- [ ]     Create PR
- [ ]     Assign reviewer(s) to PR

### PR DESCRIPTION
**Pull request by schote**
closes #3

### CODE REVIEW

File: src/mrpro/data/_QData.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+@dataclasses.dataclass(slots=True, frozen=True)
+class QData:
+    """MR quantitative data (QData) class."""
+
+    header: QHeader
+    data: torch.Tensor
+
+    @classmethod
+    def from_tensor_and_header(cls, data: torch.Tensor, header: KHeader | IHeader | QHeader) -> QData:
```
Comment (fzimmermann89) [inline src/mrpro/data/_QData.py]: From tensor and header should be an `__init__`

File: src/mrpro/data/_QData.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        """Create QData object from a tensor and an arbitrary MRpro header.
+
+        Parameters
+        ----------
+        data  # ToDo: add which dimensions?
+            torch.Tensor containing quantitative image data with dimensions (all_other, coils, z, x, y).
```
Comment (fzimmermann89) [inline src/mrpro/data/_QData.py]: order should be other,coil,z,Y,X.
torch.Tensor should not be in docstring, is in typing info.

File: src/mrpro/data/_QHeader.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    def from_dicom(cls, dicom_dataset: Dataset):
+        """Read DICOM file containing qMRI data and return QHeader object.
+
+        Parameters
+        ----------
+        dicom_dataset
+            Dataset object containing the DICOM file.
+        """
+
+        def getItems(name):
```
Comment (fzimmermann89) [inline src/mrpro/data/_QHeader.py]: maybe add a TODO to move this to utilities or reuse the logic in IHeader?

**Review by fzimmermann89 — APPROVED** [review]
Lgtm

**Comment by schuenke** [comment]
I will add the qdata from dicom feature including tests and merge it afterwards.

**Pull request merged**